lang/cs/Org.Apache.REEF.Tests/Org.Apache.REEF.Tests.csproj

```diff
@@ -67,6 +67,9 @@ under the License.
     <Compile Include="Utility\TestExceptions.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="ConfigFiles\evaluator.conf">
```

Comment:
No. As this PR is a small one, I just put back what we dropped earlier for tests.